DataFormats/Math/test/ExtVec_t.cpp

```diff
@@ -1,25 +1,21 @@
-#include "FWCore/Utilities/interface/GCC11Compatibility.h"
-#if GCC_PREREQUISITE(4,8,0)
-
 #include "DataFormats/Math/interface/ExtVec.h"
 
 #include<cmath>
 #include<vector>
 
 #include<iostream>
 
+#ifdef __AVX__
+#define CMS_USE_AVX
+#endif /* __AVX__ */
 
-// #include <x86intrin.h>
+// clang complains that it is not "standard"
+#define constexpr
```

Comment:
I find this whole constexpr business a mess.
I gave up, in any case we essentially do not use constexpr, even less for these vectors.
(at the moment we use SSEVEct on x86_64 and ExtVec on arm and ppc)
Again to be checked again with clang 4 and gcc 7